app/src/main/java/org/p2p/wallet/solend/ui/earn/SolendEarnFragment.kt

```diff
@@ -49,4 +109,9 @@ class SolendEarnFragment :
             tokensRecyclerView.isVisible = !isLoading
         }
     }
+
+    override fun onDestroyView() {
+        timerJob?.cancel()
```

Comment:
it will crash if user just presses home.
You need to pause it in onStop and cancel here

Reply:
no crash
checked right now